hack/e2e/run-rp-and-e2e.sh

```diff
@@ -102,6 +105,18 @@ run_e2e() {
     CLUSTER_SPN_SECRET=$(cat $CLUSTERSPN | jq -r .password)
 
     echo "########## 🚀 Create ARO Cluster $CLUSTER - Using client-id : $CLUSTER_SPN_ID ##########"
+    set +x
```

Comment:
I think this needs to be moved up, with `set -x` even setting the secrets will print:
```
ehashman@red-dot:~$ set -x
ehashman@red-dot:~$ SECRET="xyz"
+ SECRET=xyz
```